lib/directives/floatingUi.ts

```diff
@@ -0,0 +1,132 @@
+import type {Boundary, Placement, RootBoundary} from '@floating-ui/vue'
+export {autoUpdate} from '@floating-ui/vue'
+
+import {type DirectiveBinding, h, render} from 'vue'
+import {BPopover, BPopoverProps} from "bootstrap-vue-next";
```

Comment:
move this file in utils since it's from bootstrap vue next
credit bvn for the code